upf/quantifiers_remover.py

```diff
@@ -123,25 +138,51 @@ def _create_problem_copy_without_actions_and_goals(self):
             new_problem.set_initial_value(fl, v)
         return new_problem
 
+    def _durative_action_without_quantifiers(self, action) -> DurativeAction:
+        new_action = DurativeAction(action.name(), OrderedDict((ap.name(), ap.type()) for ap in action.parameters()), self._env)
+        new_action.set_duration_constraint(action.duration())
+
+        for t, cl in action.conditions().items():
```

Comment:
You are not using `t`, right?
```suggestion
        for cl in action.conditions().values():
```

Reply:
Actually yes, it's used in line 148. In general every time t (or i) are used when "putting back" the condition/effect in the new action

Reply:
Ah! I missed it. Ignore the comment